packages/orchestration/src/orchestration-client.ts

```diff
@@ -16,12 +16,12 @@ import type { HttpDestinationOrFetchOptions } from '@sap-cloud-sdk/connectivity'
 export class OrchestrationClient {
   /**
    * Creates an instance of the orchestration client.
-   * @param config - Orchestration module configuration.
+   * @param config - Orchestration module configuration. This can either be an instance of `OrchestrationModuleConfig` or a JSON string retrieved from AI Launchpad.
```

Comment:
```suggestion
   * @param config - Orchestration module configuration. This can either be an `OrchestrationModuleConfig` object or a JSON string retrieved from AI Launchpad.
```